### DISCUSSION TITLE
How to get intro section before projects?

### DISCUSSION THREAD

**Discussion by jamesbraza** [binokochumolvarghese/lightbi-hugo#21] [Q&A] [answered]
https://binovarghese.com/ has an intro section before going into the projects section(s).

How can one add an intro section?  The current `exampleSite` doesn't have an intro section.

**Comment by binokochumolvarghese** [comment, marked as answer]
My website is heavily customised according to my needs and preferences.

I am working on the theme to add the intro section in the home page, I hope I can realease it soon.

**Reply by heihahehe** [reply]
I have a few problems, making excuse me can share the https://binovarghese.com/ repository?
![20240101102118](https://github.com/binokochumolvarghese/lightbi-hugo/assets/152707762/1d746970-6461-4635-8442-c6e46f8d9880)

I created a new table of contents, but the title in the article page went to the left?

**Reply by binokochumolvarghese** [reply]
HI, I am sorry I can't share my repo. But you can check this demo [website ](https://github.com/binokochumolvarghese/lightbi-hugo-demo) for reference.

**Reply by heihahehe** [reply]
Thank you. That problem has been solved.
